Data-Science/Kubeflow/MNIST-Digits-Recognition/component/kubeflow-launcher-component.yaml

```diff
@@ -9,9 +9,9 @@ inputs:
 - {name: ttl Seconds After Finished,  type: Integer,    default: -1,        description: 'Defines the TTL for cleaning up finished TFJobs.'}
 - {name: CleanPodPolicy,              type: String,     default: Running,   description: 'Defines the policy for cleaning up pods after the TFJob completes.'}
 - {name: PS Spec,                     type: JsonObject, default: '{}',      description: 'TFJob ps replicaSpecs.'}
-- {name: Worker Spec,                 type: JsonObject, default: '{}',      description: 'TFJob worker replicaSpecs.'}
-- {name: Chief Spec,                  type: JsonObject, default: '{}',      description: 'TFJob chief replicaSpecs.'}
-- {name: Evaluator Spec,              type: JsonObject, default: '{}',      description: 'TFJob evaluator replicaSpecs.'}
+- {name: Worker Spec,                 type: String,     default: '{}',      description: 'TFJob worker replicaSpecs.'}
+- {name: Chief Spec,                  type: String,     default: '{}',      description: 'TFJob chief replicaSpecs.'}
+- {name: Evaluator Spec,              type: String,     default: '{}',      description: 'TFJob evaluator replicaSpecs.'}
```

Comment:
`PS Spec` should probably be also changed.

Reply:
We are not using it, but the same is true for `Evaluator Spec`

Reply:
Done